relayer/src/chain/cosmos.rs

```diff
@@ -1946,6 +1946,15 @@ impl ChainEndpoint for CosmosSdkChain {
         }
     }
 
+    fn query_host_consensus_state(&self, height: ICSHeight) -> Result<Self::ConsensusState, Error> {
+        let height = Height::try_from(height.revision_height).map_err(Error::invalid_height)?;
+
+        let response = self
+            .block_on(self.rpc_client.block(height))
```

Comment:
The plan is to replace this with the `/header` query. (I'm working on adding it to tendermint-rs -> https://github.com/informalsystems/tendermint-rs/issues/832)

Reply:
https://github.com/informalsystems/tendermint-rs/pull/1101